File: Iniciante/1017.c
#include <stdio.h>
 
int main(){ 
    float dist, lit;
    int tempo, velocidade;
	
    scanf("%d %d", &tempo, &velocidade);
    dist = tempo * velocidade;
	lit = dist / 12;
    printf("%.3lf\n", lit);

    return 0;
}

File: Iniciante/1017.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner scanf = new Scanner(System.in);

        float dist, lit;
        int tempo = scanf.nextInt();
        int velocidade = scanf.nextInt();

        dist = tempo * velocidade;
	    lit = dist / 12;

        System.out.printf("%.3f\n", lit);

    }
    
}